lib/json_schema_form/umm_preview.rb

```diff
@@ -1,13 +1,17 @@
 # :nodoc:
 class UmmPreview < JsonFile
-  attr_accessor :resource, :forms
+  attr_accessor :data, :forms, :draft_id
 
-  def initialize(schema_type:, preview_filename:, resource:)
-    super(schema_type, preview_filename)
+  # schema_type:      published_resource_name (service/variable)
```

Comment:
what's the reasoning for this to be the singular form, rather than `plural_published_resource_name`? It seems that all the other UMM classes use the plural, and you're having to call `.pluralize` most of the time `schema_type` is used, including to call `super`.

Reply:
I think it is used 4 places, 2 of them need singular a 2 need plural, I liked the idea of using the simpler version in the param.

Reply:
I do think it's a bit odd to have to change it when you're calling `super`, but I guess it might be more a matter of opinion.